Tasks/RunRemoteShellCommandsSSH/task.json

```diff
@@ -0,0 +1,89 @@
+{
+    "id": "91443475-df55-4874-944b-39253b558790",
+    "name": "RunRemoteShellCommandsSSH",
+    "friendlyName": "Remote Shell Commands",
+    "description": "Run shell commands on a remote machine using SSH.",
+    "helpMarkDown": "",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "demands": [
+     ],
+    "instanceNameFormat": "Run shell commands on remote machine using SSH",
+    "groups": [
+        {
+            "name": "connection",
+            "displayName": "Connection",
+            "isExpanded": true
+        }
+    ],
+    "inputs": [
+        {
+            "name": "hostname",
+            "type": "string",
+            "label": "Hostname",
+            "defaultValue": "",
+            "required": true,
+            "groupName": "connection",
+            "helpMarkDown": "Hostname or IP of the remote machine."
+        },
```

Comment:
Agree with Bryan, my plan is to do service endpoint anyway, so didn't do the multiple machines.